### PR TITLE
dev_guide/builds.adoc: mention that pullSecret may be used for all type of strategies

### PR DESCRIPTION
**Pull request by php-coder**
Maybe I missed something but I didn't find where we explicitly talking that `pullSecret` may be used with all of our strategies. As far I can see in the sources, we can use it.

@rhcarvalho PTAL.

### CODE REVIEW

**Comment by rhcarvalho** [comment]
@php-coder your note LGTM, and it is what we document in the REST API [1, 2, 3].

The original docs for `pullSecret` were added back in #398. @mfojtik could you please confirm `pullSecret` applies to all strategies?

[1] https://docs.openshift.org/latest/rest_api/openshift_v1.html#v1-sourcebuildstrategy
[2] https://docs.openshift.org/latest/rest_api/openshift_v1.html#v1-dockerbuildstrategy
[3] https://docs.openshift.org/latest/rest_api/openshift_v1.html#v1-custombuildstrategy

**Comment by mfojtik** [comment]
yes, it applies to all strategies.

File: dev_guide/builds.adoc [hunk 1]
```diff
@@ -1091,6 +1091,9 @@ is *dockerhub*:
 ----
 ====
 
+NOTE: We have used `*pullSecret*` for Source build but it is applicable to
+Docker and Custom builds too.
```
Comment (adellape) [inline dev_guide/builds.adoc]: Suggest:

```
[NOTE]
====
This example uses `*pullSecret*` in a Source build, but it is also applicable in Docker and Custom builds.
====
```

**Comment by adellape** [comment]
@php-coder Sorry for the delay; this now needs a rebase. Also one comment inline.

**Comment by php-coder** [comment]
@adellape Updated. PTAL.

**Comment by adellape** [comment]
:+1: Thanks.

**Pull request merged**